akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -115,12 +139,13 @@ sealed trait RequestEntity extends HttpEntity with jm.RequestEntity with Respons
  */
 sealed trait ResponseEntity extends HttpEntity with jm.ResponseEntity {
   def withContentType(contentType: ContentType): ResponseEntity
-
-  override def transformDataBytes(transformer: Flow[ByteString, ByteString, Any]): ResponseEntity
+  def withSizeLimit(maxBytes: Long): ResponseEntity
```

Comment:
See https://github.com/akka/akka/pull/18888/files#r44797512, let's link to the one with docs?